api/src/test/java/org/openmrs/obs/ImageHandlerTest.java

```diff
@@ -110,20 +108,17 @@ public void saveObs_shouldRetrieveCorrectMimetype() throws IOException {
 	
 	@Test
 	public void saveObs_shouldHandleByteArrays() throws IOException {
-		File sourceFile = new File(
-		       "src" + File.separator + "test" + File.separator + "resources" + File.separator + "ComplexObsTestImage.png");
-			
+		File sourceFile = Paths.get("src", "test", "resources", "ComplexObsTestImage.png").toFile();
+		
 		byte[] bytes = FileUtils.readFileToByteArray(sourceFile);
```

Comment:
```suggestion
		byte[] bytes = Files.readAllBytes(sourceFile);
```

I said earlier that we should only changes File/Path construction to get rid of the manual concatenation of File.separator. I would say its alright to adapt **tests** to use the newer Paths/Files helper classes on Path directly instead of converting the Path to the legacy class.